core/src/main/java/com/netease/arctic/table/BaseKeyedTable.java

```diff
@@ -170,11 +172,11 @@ public UpdateProperties updateProperties() {
 
   @Override
   public long beginTransaction(String signature) {
-    try {
-      return client.allocateTransactionId(this.tableMeta.getTableIdentifier(), signature);
-    } catch (TException e) {
-      throw new IllegalStateException("failed begin transaction", e);
-    }
+    AppendFiles appendFiles = changeTable.newAppend();
+    // TODO add summary, use fast append?
```

Comment:
I don't think fast append has any difference with append here.
BTW， we may need add some comment here to describe this impelementation.

Reply:
OK, I will add some comment